<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="5" ignored="0" total="5" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="489894" started-at="2019-12-16T12:11:55Z" finished-at="2019-12-16T12:20:05Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="489894" started-at="2019-12-16T12:11:55Z" finished-at="2019-12-16T12:20:05Z">
      <class name="Tests.Fit">
        <test-method status="PASS" signature="StartReport()[pri:0, instance:Tests.Fit@1d057a39]" name="StartReport" is-config="true" duration-ms="125" started-at="2019-12-16T12:11:55Z" finished-at="2019-12-16T12:11:55Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- StartReport -->
        <test-method status="PASS" signature="PreConditon()[pri:0, instance:Tests.Fit@1d057a39]" name="PreConditon" is-config="true" duration-ms="7211" started-at="2019-12-16T12:11:55Z" finished-at="2019-12-16T12:12:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PreConditon -->
        <test-method status="FAIL" signature="GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39]" name="GetFit" duration-ms="108103" started-at="2019-12-16T12:12:03Z" data-provider="MarketConfig" finished-at="2019-12-16T12:13:51Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[United Kingdom]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[English]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Pack]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.StringIndexOutOfBoundsException">
            <message>
              <![CDATA[String index out of range: -1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.StringIndexOutOfBoundsException: String index out of range: -1
at java.lang.String.substring(String.java:1927)
at POM.SummaryPage.GetFitOrderSubmit(SummaryPage.java:2405)
at Pages.Summary.GetFitSummaryDetails(Summary.java:24)
at Tests.Fit.GetFit(Fit.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.StringIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetFit -->
        <test-method status="PASS" signature="PostCondition(org.testng.ITestResult)[pri:0, instance:Tests.Fit@1d057a39]" name="PostCondition" is-config="true" duration-ms="909" started-at="2019-12-16T12:13:51Z" finished-at="2019-12-16T12:13:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=GetFit status=FAILURE method=Fit.GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostCondition -->
        <test-method status="PASS" signature="PreConditon()[pri:0, instance:Tests.Fit@1d057a39]" name="PreConditon" is-config="true" duration-ms="5549" started-at="2019-12-16T12:13:52Z" finished-at="2019-12-16T12:13:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PreConditon -->
        <test-method status="FAIL" signature="GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39]" name="GetFit" duration-ms="80071" started-at="2019-12-16T12:13:57Z" data-provider="MarketConfig" finished-at="2019-12-16T12:15:17Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[United States]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[EspaÃ±ol]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Pack]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Critical Back end error]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Critical Back end error
at org.testng.Assert.fail(Assert.java:96)
at POM.PaymentPage.ShippingDetails(PaymentPage.java:904)
at Pages.Payment.PaymentDetails(Payment.java:15)
at Tests.Fit.GetFit(Fit.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetFit -->
        <test-method status="PASS" signature="PostCondition(org.testng.ITestResult)[pri:0, instance:Tests.Fit@1d057a39]" name="PostCondition" is-config="true" duration-ms="935" started-at="2019-12-16T12:15:17Z" finished-at="2019-12-16T12:15:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=GetFit status=FAILURE method=Fit.GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostCondition -->
        <test-method status="PASS" signature="PreConditon()[pri:0, instance:Tests.Fit@1d057a39]" name="PreConditon" is-config="true" duration-ms="5672" started-at="2019-12-16T12:15:18Z" finished-at="2019-12-16T12:15:24Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PreConditon -->
        <test-method status="FAIL" signature="GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39]" name="GetFit" duration-ms="88339" started-at="2019-12-16T12:15:24Z" data-provider="MarketConfig" finished-at="2019-12-16T12:16:52Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Canada]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[English]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Pack]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Minor Summary Labels is Displaying expected [[Enrollment Summary, Enrollment Settings, Co-Applicant/Spouse, About You, Your Shipping/Primary Address, Your Billing Address, Your Payment Information, Terms and Conditions, Summary]] but found [[Step 4: Enter Payment Information, Billing Address, Shipping/Primary Address, Shipping Method, GetFit21 Coach]]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Minor Summary Labels is Displaying expected [[Enrollment Summary, Enrollment Settings, Co-Applicant/Spouse, About You, Your Shipping/Primary Address, Your Billing Address, Your Payment Information, Terms and Conditions, Summary]] but found [[Step 4: Enter Payment Information, Billing Address, Shipping/Primary Address, Shipping Method, GetFit21 Coach]]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at POM.SummaryPage.GetFitOrderSubmit(SummaryPage.java:2136)
at Pages.Summary.GetFitSummaryDetails(Summary.java:24)
at Tests.Fit.GetFit(Fit.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetFit -->
        <test-method status="PASS" signature="PostCondition(org.testng.ITestResult)[pri:0, instance:Tests.Fit@1d057a39]" name="PostCondition" is-config="true" duration-ms="907" started-at="2019-12-16T12:16:52Z" finished-at="2019-12-16T12:16:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=GetFit status=FAILURE method=Fit.GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostCondition -->
        <test-method status="PASS" signature="PreConditon()[pri:0, instance:Tests.Fit@1d057a39]" name="PreConditon" is-config="true" duration-ms="5481" started-at="2019-12-16T12:16:53Z" finished-at="2019-12-16T12:16:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PreConditon -->
        <test-method status="FAIL" signature="GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39]" name="GetFit" duration-ms="88594" started-at="2019-12-16T12:16:58Z" data-provider="MarketConfig" finished-at="2019-12-16T12:18:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[Canada]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[FranÃ§ais]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Pack]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Minor Summary Labels is Displaying expected [[RÃ©sumÃ© d'inscription, ParamÃ¨tres d'inscription, conjoint, Ã Propos de Vous, Votre Adresse de livraison/Adresse principale, Votre Adresse de Facturation, Votre conditions de vente, Termes et Conditions, RÃ©sumÃ©]] but found [[Ãtape 4 : Entrez vos informations de paiement, Votre Adresse de Facturation, Adresse de livraison/Adresse principale, mÃ©thode d'expÃ©dition, GetFit21 Coach]]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Minor Summary Labels is Displaying expected [[RÃ©sumÃ© d'inscription, ParamÃ¨tres d'inscription, conjoint, Ã Propos de Vous, Votre Adresse de livraison/Adresse principale, Votre Adresse de Facturation, Votre conditions de vente, Termes et Conditions, RÃ©sumÃ©]] but found [[Ãtape 4 : Entrez vos informations de paiement, Votre Adresse de Facturation, Adresse de livraison/Adresse principale, mÃ©thode d'expÃ©dition, GetFit21 Coach]]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at POM.SummaryPage.GetFitOrderSubmit(SummaryPage.java:2136)
at Pages.Summary.GetFitSummaryDetails(Summary.java:24)
at Tests.Fit.GetFit(Fit.java:70)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetFit -->
        <test-method status="PASS" signature="PostCondition(org.testng.ITestResult)[pri:0, instance:Tests.Fit@1d057a39]" name="PostCondition" is-config="true" duration-ms="902" started-at="2019-12-16T12:18:27Z" finished-at="2019-12-16T12:18:28Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=GetFit status=FAILURE method=Fit.GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostCondition -->
        <test-method status="PASS" signature="PreConditon()[pri:0, instance:Tests.Fit@1d057a39]" name="PreConditon" is-config="true" duration-ms="5820" started-at="2019-12-16T12:18:28Z" finished-at="2019-12-16T12:18:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PreConditon -->
        <test-method status="FAIL" signature="GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39]" name="GetFit" duration-ms="90329" started-at="2019-12-16T12:18:34Z" data-provider="MarketConfig" finished-at="2019-12-16T12:20:04Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[United States]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[English]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Pack]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Critical Back end error]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Critical Back end error
at org.testng.Assert.fail(Assert.java:96)
at POM.PaymentPage.ShippingDetails(PaymentPage.java:904)
at Pages.Payment.PaymentDetails(Payment.java:15)
at Tests.Fit.GetFit(Fit.java:69)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:384)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:345)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:126)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:418)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- GetFit -->
        <test-method status="PASS" signature="PostCondition(org.testng.ITestResult)[pri:0, instance:Tests.Fit@1d057a39]" name="PostCondition" is-config="true" duration-ms="929" started-at="2019-12-16T12:20:04Z" finished-at="2019-12-16T12:20:05Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=GetFit status=FAILURE method=Fit.GetFit(java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:Tests.Fit@1d057a39] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- PostCondition -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:Tests.Fit@1d057a39]" name="afterSuite" is-config="true" duration-ms="1641" started-at="2019-12-16T12:20:05Z" finished-at="2019-12-16T12:20:07Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- Tests.Fit -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
